packages/integration-test/test/scheduled/eth-input-data.test.ts

```diff
@@ -35,18 +35,18 @@ describe('ETH Fee proxy detection test-suite', () => {
     const balance = await ethInputContract.getBalance({
       ...requestData,
       currency: {
-        network: 'private',
+        network: 'rinkeby',
         type: RequestLogicTypes.CURRENCY.ETH,
         value: privateErc20Address,
       },
     });
-
+    expect(balance.error).toBeUndefined();
     expect(balance.balance).toBe('50000000000000000');
     expect(balance.events).toHaveLength(1);
     expect(balance.events[0].name).toBe('payment');
-    expect(balance.events[0].amount).toBe('1');
+    expect(balance.events[0].amount).toBe('50000000000000000');
     expect(Math.abs(declarationTimestamp - (balance.events[0].timestamp ?? 0))).toBeLessThan(5);
```

Comment:
This test is testing eth input data declarative. But we actually see an event added which has the declared amount.
@kevindavee let me know if this is ok.